File: assignment/269p3-1.R
pima <- read.csv('./data/pima2.csv', header = T)
result <- aggregate(pima['X'], list(diabetes = pima$diabetes), length)
result['rate'] <- result['X'] / sum(result['X']) * 100

print(result)

result_table <- table(pima$diabetes)
par(mfrow = c(1, 2))
barplot(result_table, col = 1:2)
pie(result_table, col = 1:2)

File: assignment/269p3-1.py
import pandas as pd
import matplotlib.pyplot as plt

pima = pd.read_csv('../data/pima2.csv')

describe = pima.groupby('diabetes')['Unnamed: 0'].describe()
describe['rate'] = pima.groupby('diabetes')['age'].describe()['count'] / sum(
    pima.groupby('diabetes')['age'].describe()['count']) * 100

print(describe)

plt.bar(describe.index, describe['count'])
plt.show()
plt.pie(describe['count'], labels=describe.index)
plt.show()
